task/clair-scan/0.1/clair-scan.yaml

```diff
@@ -9,6 +9,8 @@ metadata:
     tekton.dev/tags: "appstudio, hacbs"
   name: clair-scan
 spec:
+  description: >-
+    Scan the image vulnerabilities gotten from clair on quay.io
```

Comment:
```suggestion
    Scan the image for vulnerabilities with clair
```

Reply:
Yes, `Clair`. The vulnerabilities generated by Clair and saved on quay.io, this ask just scan it and generated a more readable report.

Reply:
Updated description to `Scans the image for vulnerabilities with clair`